sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -975,7 +975,7 @@ object SQLConf {
         "checkpoint location.")
       .stringConf
```

Comment:
How about?
```scala
.transform(_.toLowerCase(Locale.ROOT))
.checkValue(
  str => Set("min", "max").contains(str)
```

Reply:
not quite relevant with what the PR targets but okay.